<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="MyOpenGLWidget" name="SceneRect"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Scene"/>
    <addaction name="actionSave_Scene"/>
    <addaction name="actionOpen_Scene"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEntities">
    <property name="title">
     <string>Entities</string>
    </property>
    <addaction name="actionNew_Entity"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionReUndo"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuRenderView">
    <property name="title">
     <string>Render</string>
    </property>
    <widget class="QMenu" name="menuEffect">
     <property name="title">
      <string>Effect</string>
     </property>
     <addaction name="actionNormal_2"/>
     <addaction name="actionBlur"/>
     <addaction name="actionBloom"/>
    </widget>
    <addaction name="actionDeffered"/>
    <addaction name="actionNormal"/>
    <addaction name="actionDiffuse"/>
    <addaction name="actionFragg"/>
    <addaction name="actionelse"/>
    <addaction name="separator"/>
    <addaction name="actionLightScene"/>
    <addaction name="separator"/>
    <addaction name="menuEffect"/>
   </widget>
   <widget class="QMenu" name="menuCreate">
    <property name="title">
     <string>Create</string>
    </property>
    <addaction name="actionSphere"/>
    <addaction name="actionCube"/>
    <addaction name="actionCone"/>
    <addaction name="actionCylinder"/>
    <addaction name="actionPlane"/>
    <addaction name="actionPyramid"/>
    <addaction name="actionTorus"/>
    <addaction name="separator"/>
    <addaction name="actionPointLight_shpere"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuEntities"/>
   <addaction name="menuCreate"/>
   <addaction name="menuRenderView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionactionNew_Scene_Icon"/>
   <addaction name="actionactionSave_Scene_Icon"/>
   <addaction name="actionactionOpen_Scene_Icon"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QDockWidget" name="dock_hierarchy">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Hierarchy</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6"/>
  </widget>
  <widget class="QDockWidget" name="dock_inspector">
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit (Alt + f4)</string>
   </property>
  </action>
  <action name="actionNew_Scene">
   <property name="text">
    <string>New Scene</string>
   </property>
  </action>
  <action name="actionNew_Entity">
   <property name="text">
    <string>New Entity</string>
   </property>
  </action>
  <action name="actionOpen_Scene">
   <property name="text">
    <string>Open Scene</string>
   </property>
  </action>
  <action name="actionSave_Scene">
   <property name="text">
    <string>Save Scene</string>
   </property>
  </action>
  <action name="actionSave_As_Scene">
   <property name="text">
    <string>Save As Scene</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionReUndo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionactionNew_Scene_Icon">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/Icons/Empty document new.png</normaloff>:/Icons/Icons/Empty document new.png</iconset>
   </property>
   <property name="text">
    <string>actionNew_Scene_Icon</string>
   </property>
   <property name="toolTip">
    <string>New Scene</string>
   </property>
  </action>
  <action name="actionactionSave_Scene_Icon">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/Icons/Disquette.png</normaloff>:/Icons/Icons/Disquette.png</iconset>
   </property>
   <property name="text">
    <string>actionSave_Scene_Icon</string>
   </property>
   <property name="iconText">
    <string>actionSave_Scene_Icon</string>
   </property>
   <property name="toolTip">
    <string>Save Scene</string>
   </property>
  </action>
  <action name="actionactionOpen_Scene_Icon">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/Icons/Folder docs.png</normaloff>:/Icons/Icons/Folder docs.png</iconset>
   </property>
   <property name="text">
    <string>actionOpen_Scene_Icon</string>
   </property>
   <property name="iconText">
    <string>actionSave_Scene_Icon</string>
   </property>
   <property name="toolTip">
    <string>Open Scene</string>
   </property>
  </action>
  <action name="actionDeffered">
   <property name="text">
    <string>Deffered</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionDiffuse">
   <property name="text">
    <string>Albedo</string>
   </property>
  </action>
  <action name="actionFragg">
   <property name="text">
    <string>Position</string>
   </property>
  </action>
  <action name="actionelse">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionLightScene">
   <property name="text">
    <string>LightScene</string>
   </property>
  </action>
  <action name="actionSphere">
   <property name="text">
    <string>Sphere</string>
   </property>
  </action>
  <action name="actionCube">
   <property name="text">
    <string>Cube</string>
   </property>
  </action>
  <action name="actionCone">
   <property name="text">
    <string>Cone</string>
   </property>
  </action>
  <action name="actionCylinder">
   <property name="text">
    <string>Cylinder</string>
   </property>
  </action>
  <action name="actionPlane">
   <property name="text">
    <string>Plane</string>
   </property>
  </action>
  <action name="actionPyramid">
   <property name="text">
    <string>Pyramid</string>
   </property>
  </action>
  <action name="actionTorus">
   <property name="text">
    <string>Torus</string>
   </property>
  </action>
  <action name="actionPointLight_shpere">
   <property name="text">
    <string>PointLight (shpere)</string>
   </property>
  </action>
  <action name="actionNormal_2">
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionBlur">
   <property name="text">
    <string>Blur</string>
   </property>
  </action>
  <action name="actionBloom">
   <property name="text">
    <string>Bloom</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">myopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
